### PR TITLE
Consolidate discipline-related text

### PR DESCRIPTION
**Pull request by frivoal**
Addresses #418

### CODE REVIEW

**Comment by dwsinger** [comment]
OK, but I think we have (separate) consensus to de-link the guidelines...

**Comment by dwsinger** [comment]
LGTM

File: index.bs [hunk 1]
```diff
@@ -1008,10 +1007,13 @@ Individual Participation Criteria</h3>
...
 	in the W3C Patent Policy [[!PATENT-POLICY]].
 
-	The [=Director=] <em class="rfc2119">may</em> suspend or remove for cause
-	a participant in any group (including the [=AB=] and [=TAG=]),
-	where cause includes failure to meet the requirements of this process,
-	the membership agreement, or applicable laws.
+	When serious and/or repeated violations do occur,
+	including failure to meet the requirements of this process,
+	the membership agreement, or applicable laws,
+	and repeated attempts to address these violations have not resolved the situation,
```
Comment (fantasai) [inline index.bs:1013]: The allowance of repeated attempts to address violations applies to the processes. It is not necessary for all violations, and indeed should not be required for e.g. egregious violations of the law or people's physical safety. r- without adjusting this merge of two paragraphs to maintain their previous meaning.

Comment (frivoal) [reply index.bs:1013]: Good point. How about:

> When serious and/or repeated violations do occur,
> including failure to meet the requirements of this process 
> where repeated attempts to address these violations have not resolved the situation,
> or violation of the membership agreement or applicable laws,
> the [=Director=] <em class="rfc2119">may</em> take disciplinary action,
> including suspending or removing for cause
> a participant in any group (including the [=AB=] and [=TAG=]).

I think this scope it better, and to be equivalent to the original phrasing. I do wonder if egregious violations of the CEPC (which fall under "failure to meet the requirements of this process") should require *repeated* attempts to resolve the situation, or if exclusion can be more immediate, but maybe this should be open as a separate issue, to keep this one editorial.

**Comment by dwsinger** [comment]
somehow repeating repeat seems excessive to me, and I agree that there are some violations that are sufficiently egregious, or where protecting the W3C or individuals is required, that immediate action may be needed.

I think trying to describe the ramp-up here is pointless; that's a best practice, that we try to warn people and take corrective action. The point of the process is that serious or repeated violations can result in disciplinary action, up to and including...

so for _process_ text I'd remove the escalation phrase:

> When serious and/or repeated violations do occur,
> including failure to meet the requirements of this process,
> or violation of the membership agreement or applicable laws,
> the [=Director=] may take disciplinary action,
> including suspending or removing for cause
> a participant in any group (including the [=AB=] and [=TAG=]).

**Review by fantasai — DISMISSED** [review]
The currently-proposed phrasing of the paragraph "When serious ..." changes the way the topic of that paragraph is perceived, and links it too closely with the CEPC, losing its more general sense of application from the original wording, because that is the closest thing mentioned that can be "violated". r-

**Comment by dwsinger** [comment]
To make it flow better, could we simply insert before

> Participants in any W3C activity `must` abide by the terms and spirit of the [CEPC]…

Participants `must` abide by the requirements of the membership agreement, this process, and applicable laws.

(I wondered about saying "particularly the CEPC" but I think it's better not singled out with such a word; gross violations of other policies would also be bad.)

This would put the word "violations" clearly into the context of all these requirement `must`s.

**Comment by dwsinger** [comment]
Does this also address #312 ?

**Comment by koalie** [comment]
> Addresses #418

Thanks @frivoal!

Yes, #432 addresses #418.
However, I’d like it if we were able to reference in the Process the document on suspending/removing participants that W3M is working on as a Chairs guidebook addition. @plehegar , as Guidebook owner, has the ball to merge it (I've let him know about the process CG call tomorrow that has this PR on the agenda), and doing so will create <https://www.w3.org/Guide/process/banning.html> (at the moment, it is staged in <https://raw.githack.com/w3c/Guide/koalie-patch-1/process/banning.html>)

(It may be that I'm able to propose the above as a commit to this PR before the process CG call, depending on whether PLH has approved it.)

Note on dependency: Given that one of the changes balloted in #432 removes the link to the out-dated document “Guidelines for Disciplinary Action” (which the AB had a weak consensus to unlink from the process) and that document is in our Chairs guidebook and is probably bookmarked by people, if #432 is approved, I’m going to raise an issue in the guidebook repo to install a redirect from it to the banning document, when process 2021 is operational.

**Comment by TzviyaSiegman** [comment]
From a CEPC perspective, I am fine with the proposed language. @koalie I am concerned with several of the details in the proposed guidelines. Where should I comment about that?

**Comment by koalie** [comment]
Hi @TzviyaSiegman 

> I am concerned with several of the details in the proposed guidelines. Where should I comment about that?

Please, take this up as quickly as you can with w3mreq@ as it's a W3C management developed document. W3M has approved that document already and we're literally ready to share it with the Members and Chairs.

File: index.bs [hunk 2]
```diff
@@ -1008,10 +1007,12 @@ Individual Participation Criteria</h3>
 	“Disclosure”
 	in the W3C Patent Policy [[!PATENT-POLICY]].
 
-	The [=Director=] <em class="rfc2119">may</em> suspend or remove for cause
-	a participant in any group (including the [=AB=] and [=TAG=]),
-	where cause includes failure to meet the requirements of this process,
-	the membership agreement, or applicable laws.
+	When serious and/or repeated violations do occur,
+	including failure to meet the requirements of this process,
```
Comment (fantasai) [inline index.bs:1011]: <ins>such as</ins><del>including</del> failure to meet the requirements of this process<del>,</del>

File: index.bs [hunk 3]
```diff
@@ -1008,10 +1007,12 @@ Individual Participation Criteria</h3>
 	“Disclosure”
 	in the W3C Patent Policy [[!PATENT-POLICY]].
 
-	The [=Director=] <em class="rfc2119">may</em> suspend or remove for cause
-	a participant in any group (including the [=AB=] and [=TAG=]),
-	where cause includes failure to meet the requirements of this process,
-	the membership agreement, or applicable laws.
+	When serious and/or repeated violations do occur,
```
Comment (fantasai) [inline index.bs:1010]: <del>When</del><ins>If</ins>

File: index.bs (hunk 2 above)
Comment (frivoal) [reply index.bs:1011]:
```suggestion
	such as failure to meet the requirements of this process,
```

File: index.bs (hunk 3 above)
Comment (frivoal) [reply index.bs:1010]:
```suggestion
	If serious and/or repeated violations do occur,
```

File: index.bs [hunk 4]
```diff
@@ -328,12 +328,8 @@ Rights of Members</h4>
 	</ul>
 
 	The rights and benefits of W3C membership are contingent upon conformance to the processes described in this document.
-	The vast majority of W3C Members faithfully follow the spirit as well as the letter of these processes.
-	When serious and/or repeated violations do occur,
-	and repeated attempts to address these violations have not resolved the situation,
-	the [=Director=] <em class="rfc2119">may</em> take disciplinary action.
-	Arbitration in the case of further disagreement is governed by paragraph 19 of the Membership Agreement [[MEMBER-AGREEMENT]].
-	Refer to the <a href="https://www.w3.org/2002/09/discipline">Guidelines for Disciplinary Action</a> [[DISCIPLINARY-GL]].
+	Arbitration is governed by paragraph 19 of the Membership Agreement [[MEMBER-AGREEMENT]].
```
Comment (wseltzer) [inline index.bs:331]: delete: "Arbitration is governed by paragraph 19 of the Membership Agreement [[MEMBER-AGREEMENT]]." as it doesn't add anything here.

**Review by wseltzer — CHANGES_REQUESTED** [review]
To cut unnecessary words, remove more of the holdover from previous, as it just adds another pointer to Member Agreement.

**Review by tantek — CHANGES_REQUESTED** [review]
The interspersed diffs & comments for this file are a bit of a confusing mess. Could someone do what is needed to resolve all the comments so we can actually review the effective diff?

**Comment by frivoal** [comment]
@tantek, looking at this through github's "conversation" view *is* a bit of a mess, but if you look at [the "files changed" view](https://github.com/w3c/w3process/pull/432/files), the whole diff is there, with 3 suggestions for changes inline (2 from @fantasai, 1 from @wseltzer). Nevertheless, @TzviyaSiegman +1ed your comment, and @jeffjaffe made largely the same remark during the call, so if multiple reasonable people have the same feeling, there's probably something to it.

I think the 2 suggestions from @fantasai are completely editorial and just make the phrasing better, but I initially didn't want to merge them without the group's input, in case someone thought they affected the meaning in an undesirable way. Given that nobody so far has complained about the substance about those edits, but that multiple people are unhappy about having a multi-layered thing to review, I'm now merging them.

As for @wseltzer 's comment, personally I'd be inclined to reject it, but it doesn't feel right for me do to this on my own accord. My opinion does not weight more than @wseltzer 's. That said, the sentence @wseltzer wants to remove exists prior to this clean up, so we can disentangle this by dropping her suggestion **from this PR**, and to filing it separately. @wseltzer, assuming your support for this PR doesn't hinge on that, and that this is just one more thing you though we could/should do, do you mind if I refile it as a separate issue?

**Comment by dwsinger** [comment]
(Incidentally, I am not terribly fussed about the difference between "when X occurs, Y happens" and "if X occurs, Y happens".  One emphasizes the temporality, the other the conditionality; I slightly preferred the former for some reason, but whatever)

**Review by wseltzer — APPROVED** [review]

**Pull request merged**